### ISSUE TITLE
kak-lsp opens a scratch buffer

### ISSUE THREAD

**Issue by mreppen** [kak-lsp/kak-lsp#593]
When opening a file using `kak filename`, an extra `*scratch*` buffer is opened when kak-lsp is enabled. Without, only the file buffer is opened. This happens with a kakrc that only enables kak-lsp and otherwise not.

Version: 12.0.1

**Comment by krobelus** [comment]
The scratch buffer might come from c3aeb0da (Detect wcwidth(3) sanity by querying
%val{cursor_display_column}, 2022-01-22) though I can't reproduce the problem.
That buffer should be deleted before handing back to the user's kakrc (i.e. before lsp-enable is called)

This test works for me

```sh
#!/bin/sh

. test/lib.sh

cat > filename << EOF
EOF

test_tmux_kak_start filename
test_tmux send-keys ':echo %val{buflist}' Enter
test_sleep
test_tmux capture-pane -p | tail -n1
# CHECK: *debug* *scratch* filename {{.*}}
```

**Comment by krobelus** [comment]
sorry I understand the issue now, apparently I wasn't fully awake when I looked at this. Fixed as of 6d38d186
